tslint.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
If you're using `gts` for lint/style, I believe that's based on eslint and doesn't use tslint at all?